app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Post for our fakebook app
+class Post < ApplicationRecord
+  validates :user, presence: true
```

Comment:
If you have the associations between `User` and `Post` you don't need to validate the user presence when you create posts like `u.posts.build(params)`